OSDOCS-16939: DITA abstracts — OLM understanding topics (#115281)
openshift/openshift-docs: modules/olm-installplan.adoc

[id="olm-installplan_{context}"]
= Install plan

[role="_abstract"]

> An install plan, defined by an `InstallPlan` object, describes the set of resources that Operator Lifecycle Manager (OLM) creates to install or upgrade to a specific Operator version on {product-title}. A subscription creates an install plan that you can approve automatically or manually before OLM installs the Operator.

Reviewer: ```suggestion
An _install plan_, defined by an `InstallPlan` object, describes a set of resources that Operator Lifecycle Manager (OLM) creates to install or upgrade to a specific version of an Operator. The version is defined by a cluster service version (CSV).
```
